### ISSUE TITLE
Add an `fyi` flag

### ISSUE THREAD

**Issue by tylerl0706** [eslint/eslint#6931]
Hi all,

I'm coming from the perspective of someone that has a huge-ish product already developed but wants to use ESLint moving forward. We want to see the "errors & warnings" that ESLint generates, but we don't want our npm script to error out when it happens.

We thought about ignoring whole files, but we still want to see the warnings and errors that are generated.

Enter the `fyi` flag or "for your information" for those of you that don't know. It can display the errors and warnings of ESLint, but won't error out. With npm scripts, it might look something like this: `npm check-code-style && npm start` where check-code-style could run ESLint with the `--fyi` flag. Then, even if errors are present, the app still starts.

I've always found that with ESLint, you either start developing something with it... or you don't. Adding it in after the fact is often tough being buried with errors and warnings that have to be resolved right this second or else you can't continue to develop your app. An `--fyi` flag would help show the user what they're doing wrong without stopping them from continuing their work.

Cheers!

**Comment by ilyavolodin** [comment]
Does npm script like that solve your problem?

```sh
eslint . ; exit 0
```

This will still report all of the errors, but will always exit with success error code.

**Comment by tylerl0706** [comment]
@ilyavolodin That actually does the trick! Glad it was simpler than I thought. Thanks.
